lib/node_modules/@stdlib/blas/base/sdot/lib/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* BLAS level 1 routine to tade the dot product of vectors `x` and `y`.
+*
+* @module @stdlib/blas/base/sdot
+*
+* @example
+* var sdot = require( '@stdlib/blas/base/sdot' );
```

Comment:
Typically, we order the `require` statements such that requiring the current package is the last `require` statement.